app/routing/VersionRoutingMap.scala

```diff
@@ -34,9 +37,17 @@ trait VersionRoutingMap {
 }
 
 // Add routes corresponding to available versions...
-case class VersionRoutingMapImpl @Inject()(defaultRouter: Router, v1Router: v1.Routes) extends VersionRoutingMap {
+case class VersionRoutingMapImpl @Inject()(appConfig: AppConfig,
+                                           defaultRouter: Router,
+                                           v1Router: v1.Routes,
+                                           v1RouterWithMarriageAllowance: v1WithMarriageAllowance.Routes) extends VersionRoutingMap {
+
+  val featureSwitch: FeatureSwitch = FeatureSwitch(appConfig.featureSwitch)
 
   val map: Map[String, Router] = Map(
-    VERSION_1 -> v1Router
+    VERSION_1 -> {
+      Logger.apply("").logger.error(featureSwitch.isMarriageAllowanceRoutingEnabled.toString)
```

Comment:
Few points:

- It's unclear why this logging is occurring as part of the mapping for `VERSION_1`. This line should be outside of this map val (but in the class constructor after `featureSwitch` is defined)
- Have the class extend the `Logging` trait and use `logger` instead of creating a fresh `Logger` with an empty name.
- The error message should probably say something more than just "true" or "false"?

Reply:
Also should this be at ERROR level? I get that we want to see it but would WARN do?

Reply:
whoopsie, that was a test log that I forgot to remove